src/PhpWord/Element/AbstractElement.php

```diff
@@ -108,13 +108,27 @@
     protected $mediaRelation = false;
 
     /**
-     * Is part of collection; true for Title, Footnote, Endnote, and Chart
+     * Is part of collection; true for Title, Footnote, Endnote, Chart, and Comment
      *
      * @var bool
      */
     protected $collectionRelation = false;
 
     /**
+     * The start position for the linked comment
+     * 
+     * @var Comment
+     */
+    protected $commentStart;
```

Comment:
@rvanlaak Indeed, the Comment starts at an element and ends at another one.

The node in the generated XML is named 'w:commentRangeStart and 'w:commentRangeEnd, maybe I could rename it to match this closer.

On the Comment itself, I'm keeping a reference back to the Element on which it starts/ends
 private $startElement;
 private $endElement;